data-collection/deploy/module-aws-feeds.yaml

```diff
@@ -111,6 +111,17 @@ Resources:
           from dateutil.parser import parse
           import boto3
 
+          FEEDS_MAP = {
```

Comment:
May be a good idea to move it to the Lambda environment variable to keep all config there, otherwise there are 2 places to control it (hardcoded here and env var)

Reply:
json in env vars? I would not do that. Env vars are good to control feature flag type of configuration but not for such big as prefix+url imo